src/ui/public/index_patterns/__tests__/_index_pattern.js

```diff
@@ -405,12 +404,14 @@ describe('index pattern', function () {
 
   describe('#isWildcard()', function () {
     it('returns true if id has an *', function () {
-      indexPattern.id = 'foo*';
-      expect(indexPattern.isWildcard()).to.be(true);
+      return create('foo*').then(function (pattern) {
```

Comment:
A rejected promise wouldn't run the assertion, true, but it would also fail the test.